src/Runner.Common/JobServer.cs

```diff
@@ -169,6 +170,10 @@ private void InitializeWebsocketClient(TimeSpan delay)
                     Trace.Info($"Creating websocket client ..." + feedStreamUrl);
                     this._websocketClient = new ClientWebSocket();
                     this._websocketClient.Options.SetRequestHeader("Authorization", $"Bearer {accessToken}");
+                    var userAgent = HostContext.UserAgents;
+                    var userAgentStrings = userAgent.Select(x => x.ToString());
```

Comment:
I know I am atleast missing `UserAgentUtility.GetDefaultRestUserAgent` as a prefix. There is also suffix, not sure where it is from...

```
VSServices/2.289.2.0 (NetStandard; Darwin 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64) GitHubActionsRunner-osx-x64/2.289.2 ClientId/ac8320a3-2b5d-4224-b9ac-36aed664839b RunnerId/65 GroupId/1 CommitSHA/4145a8c2bb76a25927acd779977a4f9faa393666 (Darwin 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64)
```

```
UserAgentUtility.GetDefaultRestUserAgent HostContext.UserAgents ???
```

Reply:
wondering if there a better way here/ single place to get the blob from or should I create one?